avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1660,7 +1668,7 @@ def splitPart(cfg, particles, dem):
     return particles
 
 
-def savePartToPickle(dictList, outDir):
+def savePartToPickle(dictList, outDir, logName):
```

Comment:
Should we also adapt the read particles from pickle function with the logName and the write particle from com1DFA?

Reply:
made an issue